oss.krtirtho.Spotube.yml

```diff
@@ -0,0 +1,51 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/master/data/flatpak-manifest.schema.json
+app-id: oss.krtirtho.Spotube
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+rename-desktop-file: spotube.desktop
+rename-appdata-file: spotube.appdata.xml
+command: spotube
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --filesystem=~/.local/share/oss.krtirtho.spotube:create
```

Comment:
are you sure?

This looks like the default `XDG_DATA_HOME` that is actually different inside a flatpak.

If anything (ie the application doesn't follow XDG_DATA_HOME) it should be `--persist`

Reply:
After investigating a bit I found I don't need access to this at all. The plugin that I use which uses this path or similar path can work inside the flatpak sandbox too. But I do need access for the `xdg-download` folder. I'll be adding this in the next commit